### PR TITLE
1lab

### PR DESCRIPTION
**Pull request by Tourie**

### CODE REVIEW

File: Converter/app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    }
+    compileSdkVersion 30
+    buildToolsVersion "30.0.2"
+
+    defaultConfig {
+        applicationId "com.example.converter"
+        minSdkVersion 16
+        targetSdkVersion 30
+        versionCode 1
+        versionName "1.0"
```
Comment (gmltA) [inline Converter/app/build.gradle]: No APK versioning. See requirements.

**Review by gmltA — CHANGES_REQUESTED** [review]
Your files contain identical code. Use inheritance to avoid code duplication. Fragments are useless in your project. You should handle view interaction in related fragment. Not in the activity.

**Comment by Tourie** [comment]
I think, now it's looking better

**Review by gmltA — APPROVED** [review]
A-ha. Now it's better.

**Pull request merged**